pkg/operator/collection.go

```diff
@@ -151,6 +151,7 @@ func (r *collectionReconciler) ensureCollectorSecrets(ctx context.Context, spec
 			Namespace: r.opts.OperatorNamespace,
 			Labels: map[string]string{
 				LabelAppName: NameCollector,
+				KubernetesAppName: CollectorAppName,
```

Comment:
You shouldn't need this on the secret - just pods.

Reply:
done.